### PR TITLE
Fix issue #518 : Add delay for group requests

### PR DESCRIPTION
**Pull request by Bari77**
Add a delay of 5 seconds on group request

### CODE REVIEW

**Comment by 0Lucifer0** [comment]
may be worth adding tests on this

**Comment by erwan-joly** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/NosCore.Controllers/Group/PjoinPacketHandler.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/NosCore/NosCore/pullRequest?prid=4511083&bid=15200050)

**Comment by 0Lucifer0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/NosCore.Controllers/Group/PjoinPacketHandler.cs  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/NosCore/NosCore/pullRequest?prid=4511083&bid=15200050)

**Comment by 0Lucifer0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/NosCore.Controllers/Group/PjoinPacketHandler.cs  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/NosCore/NosCore/pullRequest?prid=4511083&bid=15200050)

**Comment by 0Lucifer0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/NosCore.Controllers/Group/PjoinPacketHandler.cs  4
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/NosCore/NosCore/pullRequest?prid=4511083&bid=15200050)

File: src/NosCore.Controllers/Group/PjoinPacketHandler.cs [hunk 1]
```diff
@@ -89,7 +91,7 @@ public override void Execute(PjoinPacket pjoinPacket, ClientSession clientSessio
                     }
 
                     var blacklisteds = _blacklistHttpCLient.GetBlackLists(clientSession.Character.VisualId);
-                    if (blacklisteds.Any(s => s.CharacterId == pjoinPacket.CharacterId))
+                    if (blacklisteds != null && blacklisteds.Any(s => s.CharacterId == pjoinPacket.CharacterId))
```
Comment (erwan-joly) [inline src/NosCore.Controllers/Group/PjoinPacketHandler.cs]: blacklisted should never be null

**Pull request merged**